content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,81 +118,140 @@ On your validator:
 
-5. Configure your validator host machine's firewall to allow the following traffic only:
+4. Configure your validator host machine's firewall to allow the following traffic only:
 
     - Inbound traffic: Only from IP addresses of the stock `rippled` servers in the cluster you configured.
 
     - Outbound traffic: Only to the IP addresses of the stock `rippled` servers in the cluster you configured and to <https://vl.ripple.com> through port 443.
 
-6. Restart `rippled`.
+5. Restart `rippled`.
 
         $ sudo systemctl restart rippled.service
 
-7. Use the [Peer Crawler](peer-crawler.html) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
+6. Use the [Peer Crawler](peer-crawler.html) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
+
+<!-- { TODO: Future: add a recommended network architecture diagram to represent the proxy, clustering, and firewall setup: https://ripplelabs.atlassian.net/browse/DOC-2046 }-->
+
+
+### Connect using public hubs
+
+This configuration connects your validator to the network using two public hubs. This configuration is similar to [connecting using proxies you run yourself](#connect-using-proxies), but instead you connect through public hubs. Successful public hubs embody the following traits:
+
+- Good bandwidth.
+
+- Connections with a lot of reliable peers.
+
+- Ability to relay messages reliably.
+
+**Pros:**
+
+  - Enables you to trust your validator's peers because you only connect to well-known servers with a high reputation.
+
+  - Provides easy access to safe connections to the network.
+
+  - Compared to [connecting using proxies](#connect-using-proxies), may be less likely to cause your validator to disconnect from the network due to a simultaneous peer outage.
+
+**Cons:**
+
+  - Uses the default (most popular) public hubs, so it's distinctly possible that a public hub may be too busy to provide your validator with a connection to the network. Assuming that you can solely rely on a default public hub to provide your validator with a connection to the network may not be the best idea.
+
+    To help avoid this issue, use more public hubs; the more the better. It can also help to use non-default hubs, which are less likely to be busy helping new servers find other peers.
+
+_**To connect your validator to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file:**_
```

Comment:
It would be nice to have these instructions follow a parallel structure with the instructions for configuring w/ peers. That probably means moving "edit the config file" to be within the steps, then adding a step to restart the rippled service at the end.